### DISCUSSION TITLE
Windows config file path

### DISCUSSION THREAD

**Discussion by mangkoran** [wezterm/wezterm#3195] [Q&A] [answered]
Where should I put wezterm's config file in Windows? I have read [the documentation](https://wezfurlong.org/wezterm/config/files.html) but Windows doesn't have `$HOME`. Well technically I could put the config file in the same directory as `wezterm.exe`, however the executable is located in `C:\Program Files\WezTerm`.

Related https://github.com/wez/wezterm/discussions/2909, however I don't use MSYS.

Usually for Windows app, the config file (or User-specific data generally) are located in `%APPDATA%`.

**Comment by edshamis** [comment, marked as answer]
$HOME is a user dir, like C:/Users/{user}/
You put the config file in C:/Users/{user}/.config

**Reply by mangkoran** [reply]
Shoot I should've noticed it. Thank you!

**Comment by wez** [comment]
As the docs say, I recommend that you put `.wezterm.lua` in your home directory on all operating systems.  On my windows system, that is `C:/Users/wez/.wezterm.lua`.

**Reply by mangkoran** [reply]
Thank you for your reply!
